doc/2_loading.md

```diff
@@ -39,6 +39,22 @@ rdd.toArray.foreach(println)
 // CassandraRow{word: foo, count: 20}   
 ```
 
+### Using fake CassandraRDD implementation
+
+In some cases there may be a need to use an empty RDD which is an instance of `CassandraRDD` but it
+does not perform any validation for the existence of the provided table and columns and it does not
+even try to fetch the results from the database. Such an RDD can represent a table which does not
+exists. To create one, just initialize `CassandraRDD` as usual and then call `toEmptyCassandraRDD`
+method on it. The method will return the `EmptyCassandraRDD` instance.
+
+Example:
+
+```scala
+// validation is deferred, so it is not triggered during rdd creation
+val rdd = sc.cassandraTable[SomeType]("ks", "not_existing_table")
+val emptyRDD = rdd.toEmptyCassandraRDD
+```
```

Comment:
Good catch for these methods. I think I'll remove the `EmptyCassandraRDD` class and make a param which will denote whether the RDD is empty or not.

For the first comment - this empty RDD is supposed to be CassandraRDD, which includes such properties as keyspace and table name. 
I'll provide a method to create it directly but I think that the user still needs to provide keyspace and table names, even if they do not exists.